src/batch-client/index.ts

```diff
@@ -566,6 +567,92 @@ export class ZimbraBatchClient {
 			return normalized;
 		});
 
+	public searchConversation = ({
+		allowableTaskStatus,
+		calExpandInstEnd,
+		calExpandInstStart,
+		conversationId,
+		cursor,
+		fetch,
+		field,
+		fullConversation,
+		header,
+		html,
+		includeTagDeleted,
+		includeTagMuted,
+		inDumpster,
+		limit,
+		locale,
+		max,
+		needExp,
+		nest,
+		neuter,
+		offset,
+		prefetch,
+		query,
+		quick,
+		read,
+		recip,
+		resultMode,
+		sortBy,
+		timezone,
+		types,
+		wantContent
+	}: SearchConversationOptions) => {
+		let body: any;
+		body = {
+			cid: conversationId,
+			...{
```

Comment:
do we need this line, as it creates object and destructures it, we can remove it entirely